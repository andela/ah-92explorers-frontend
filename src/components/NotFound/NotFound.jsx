
import React from 'react';

export const NotFound = () => (
  <div id="notfound">
    <div className="notfound">
      <div className="notfound-404">
        <h1>Oops!</h1>
        <h2>404 - The Page cant be found</h2>
      </div>
      <a href="/">Go TO Homepage</a>
    </div>
  </div>
);
