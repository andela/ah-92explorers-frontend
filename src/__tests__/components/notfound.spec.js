import '@babel/polyfill';
import React from 'react';
import { shallow } from 'enzyme';
import { NotFound } from '../../components/NotFound/NotFound';

describe('Render NotFound Component', () => {
  it('to have wrapper class', async () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
