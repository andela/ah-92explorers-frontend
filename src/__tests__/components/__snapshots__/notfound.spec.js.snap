// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render NotFound Component to have wrapper class 1`] = `ShallowWrapper {}`;
